apps/backend/src/modules/users/user.repository.ts

```diff
@@ -47,10 +49,22 @@ class UserRepository implements Repository {
 			.query()
 			.findById(id)
 			.whereNull("deletedAt")
-			.returning("*")
-			.execute();
+			.withGraphFetched("groups.permissions")
+			.modifyGraph("groups", (builder) => {
+				builder.select("name");
+			})
+			.modifyGraph("groups.permissions", (builder) => {
+				builder.select("name", "key");
+			})
```

Comment:
we need to include group id and permission id, just best practice to return identifier for all resources